File: CodeWars/7kyu/mumbling/main.js
// My Method 1~ 🦉
function accum(s) {
    const array = s.split("");
    const new_array = [];
    for (let i = 0; i < array.length; i++) {
        new_array.push(
            array[i].toUpperCase() + array[i].repeat(i).toLowerCase()
        );
    }
    return new_array.join("-");
}

const test_case = accum("RqaEzty");
console.log(test_case);

File: CodeWars/7kyu/mumbling/main.py
# My Method 1~ 🦉
def accum(s):
    return "-".join([(letter * i).title() for i, letter in enumerate(s, 1)])


test_cases = ["abcd", "RqaEzty", "cwAt"]

for test in test_cases:
    test_case = accum(test)
    print(test_case)
